kolibri/core/webpack/middleware.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from django.shortcuts import render
+
+from .hooks import WebpackError
+
+
+logger = logging.getLogger(__name__)
+
+
+class WebpackErrorHandler:
+    def __init__(self, get_response):
+        self.get_response = get_response
```

Comment:
Could we get this to only run if DEVELOPER_MODE is set to `True`?

Can do this by simply raising a `MiddlewareNotUsed` exception during the `__init__` if not: https://docs.djangoproject.com/en/1.11/topics/http/middleware/#marking-middleware-as-unused

Alternatively, we can only add this in the dev settings, not in the production settings.